### DISCUSSION TITLE
What's the difference between the qt5 windows version and the non qt5 one?

### DISCUSSION THREAD

**Discussion by Context6016** [Martchus/syncthingtray#241] [Q&A] [answered]
And which one should I use?

**Comment by Martchus** [comment, marked as answer]
You most likely want to use the version without "-qt5"-suffix. The versions without "-qt5"-suffix are using the latest version of the underlying Qt and the versions with "-qt5"-suffix are still using the older version Qt 5. I only provide the latter for compatibility with older Windows versions. Checkout the relevant [README section](https://github.com/Martchus/syncthingtray/blob/master/README.md#packages-and-binaries) for details.
